modules/packages/LinearAlgebra.chpl

```diff
@@ -927,6 +927,25 @@ proc trace(A: [?D] ?eltType) {
   return trace;
 }
 
+proc kroneckerProduct(A: [?ADom] ?eltType, B: [?BDom] eltType) {
+  if ADom.rank != 2 || BDom.rank != 2 then compilerError("Rank size not 2");
+  const (rowA, colA) = A.shape;
+  const (rowB, colB) = B.shape;
+  var C = Matrix(rowA*rowB, colA * colB);
+  for i in 0..rowA-1 {
```

Comment:
Try the count operator for these, e.g.

```chpl

for i in 0..#rowA {
  ...
}
```

Reply:
Also, a `forall` seems appropriate here, since each iteration can be computed independently.

Reply:
yes, I was thinking of it too, forall may give better performance